src/infra/alarm/template.yaml

```diff
@@ -623,3 +623,83 @@ Resources:
       Dimensions:
         - Name: service
           Value: InterventionsProcessorFunction
+
+#
+# Cloudwatch alarms for Status Retriever
+#
+
+  AccountNotFound:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: "The queried account cannot be found"
+      Namespace: !Ref SystemTagValue
+      MetricName: ACCOUNT_NOT_FOUND
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      Statistic: Sum
+      Threshold: 1
+      EvaluationPeriods: 5
+      Period: 60
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: service
+          Value: StatusRetrieverFunction
```

Comment:
Is this an alarm for every GET status request that happens for a user that's not in the AIS table? If so, then I don't think we should have an alarm for this - I imagine most users will never have an intervention on their account so won't be in the AIS table.

Reply:
Yeah it is, I was dubious about adding it in, I'm happy to remove it.